site/content/overview/gateway-api-compatibility.md

```diff
@@ -132,63 +104,85 @@ Fields:
       - `Conflicted/True/ProtocolConflict`
       - `Conflicted/False/NoConflicts`
 
+---
+
+### GatewayClass
```

Comment:
Can we keep GatewayClass as the first resource in the doc?

Reply:
@kate-osborn We can, though I specifically changed it to match the order of the API specification:

https://gateway-api.sigs.k8s.io/reference/spec/

We direct people to it at the top of the page, so if someone is scrolling down both pages concurrently our documentation suddenly diverges for this one specific instance. 

We don't need to justify doing things slightly differently, but it's a minor concern for user experience someone might verbalise at a later date.

Reply:
Ahh, I see. That makes sense!